platform/platform-configuration-migration/src/test/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigrationTest.java

```diff
@@ -622,13 +580,12 @@ void moveFile(Path source, Path destination) throws IOException {
     @Test
     public void testNotifyWithValidFile() throws Exception {
```

Comment:
I guess things are confusing here because we set an expectation for `FileUtils.deleteQuietly()` to return `true` and are passing in `Paths.get("/root/etc/failed/pid.config")` to the `notify()` method. @figliold, do you think changing these would make the test a little clearer?